### DISCUSSION TITLE
Excuse me，how to make header has color

### DISCUSSION THREAD

**Discussion by tomuGo** [fluentribbon/Fluent.Ribbon#1025] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24789356/167146305-6e9f7c7a-b9fe-4060-98f8-0dc819e6ac47.png)
How to make here have a color？
thanks！
wish you a happy life！

**Comment by batzen** [comment, marked as answer]
You have to define a brush resource (either global if you want to change it for all group boxes or local to the control for just a single one) with the key Fluent.Ribbon.Brushes.RibbonGroupBox.Header.Foreground.
